src/bot.py

```diff
@@ -6,7 +6,7 @@
 @description: Bot that posts a tiny flower bed on Twitter every few hours.
 '''
 
-import tweepy
+# import tweepy

```

Comment:
Why are you commenting out the `tweepy` module?

Reply:
My bad forgot to uncomment `tweepy`, will resolve.